packages/manager/apps/pci-kubernetes/src/pages/new/steps/LocationStep.component.tsx

```diff
@@ -127,9 +127,11 @@ export function LocationStep({
 
   return (
     <>
-      <div className="my-4">
-        <Notifications />
-      </div>
+      {regionError && (
```

Comment:
Does this mean that notification on the `addSuccess` (line 100) won't be shown anymore?